prober/http.go

```diff
@@ -243,6 +330,38 @@ func ProbeHTTP(ctx context.Context, target string, module config.Module, registr
 	if resp == nil {
 		resp = &http.Response{}
 	}
+	var (
+		earlierstCertExpiry time.Time
+		i                   = 0
+	)
+	for _, trace := range tt.traces {
+		// We get the duration for the first request from chooseProtocol
```

Comment:
It's missing that from here then, so isn't giving a full view

Reply:
Not sure what you mean. At this point we already recorded the time to resolve the first redirect in chooseProtocolMetrics, so that should be right, no?